packages/gatsby/cache-dir/polyfill-entry.js

```diff
@@ -0,0 +1,5 @@
+import "gatsby-legacy-polyfills"
+
+if (process.env.NODE_ENV === `development`) {
```

Comment:
Isn't this file only included in `cache-dir/develop-static-entry` and so isn't this condition always true?

Reply:
this is used for production too